setup.py

```diff
@@ -50,7 +50,10 @@
         'fastparquet',
         'ragas',
         'datasets',
-        'lxml'
+        'lxml',
+        'tiktoken',
+        'spacy',
+        'nltk',
```

Comment:
I think installation of spacy and nltk is optional for many users. 
Just check installation spacy and nltk when they try to use corresponding text splitters. 
I think langchain's text splitter already support that feature. So it is okay you to just delete spacy and nltk at setup.py